app/modules/asset_groups/resources.py

```diff
@@ -0,0 +1,363 @@
+# -*- coding: utf-8 -*-
```

Comment:
Despite my efforts to persuade it otherwise, gitlab has decided this file is entirely new. It's not, it's just name changes